frontend/src/index.d.ts

```diff
@@ -1,2 +1,20 @@
 // declare module '@globalhive/vuejs-tour'
-declare module 'vue3-json-viewer';
\ No newline at end of file
+declare module 'vue3-json-viewer' {
+    import { AllowedComponentProps, App, Component, ComponentCustomProps, VNodeProps } from 'vue'
+    interface JsonViewerProps {
+        value: Object | Array<any> | string | number | boolean; //对象
+        expanded: boolean; //是否自动展开
+        expandDepth: number; //展开层级
+        copyable: boolean | object; //是否可复制
```

Comment:
@bradleySuira what are this comments?? :D

Reply:
fixed, we ended using a portion of a package/component because was not able to extend and don't want to create our own from scratch